src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -16,16 +32,180 @@ interface ComponentProps {
     title: string;
     startDate?: string;
     endDate?: string;
+    activeProject: number;
+    analysisId: number;
+    setAnalysisDeleteId: (value: number) => void;
+}
+
+interface AnalysisPillars {
+    id: number;
+    assigneeName: string;
+    title: string;
+    mainStatement: string;
+    informationGap: string;
+    filters?: string;
```

Comment:
Are we sure `filters` is string?

Reply:
I'm not sure what the filter type is yet because the work has not been done in server side yet. But I  think it is similar to the filters used in Analysis Framework. For now, it's just a null field. What should I do?